static/test/javascripts-legacy/helpers/svg-paths.js

```diff
@@ -0,0 +1,45 @@
+define(function() {
```

Comment:
Just in case someone comes up with a new SVG, is it worth mentioning this file in the `svgs` module?